File: beginner/javascript/1961.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var p, q, cnt = 0, i, dif;

var valores = lines[0].split(" ");

p = parseInt(valores[0]);
q = parseInt(valores[1]);

var jump = lines[1].split(" ").map(Number);

for(i = 1; i < q; i++){
    if(jump[i] > jump[i-1])
        dif = (jump[i] - jump[i-1]);
    else
        dif = (jump[i-1] - jump[i]);
    if(dif <= p)
        cnt++;
}

if(cnt === q - 1)
    console.log("YOU WIN");
else
    console.log("GAME OVER");

File: beginner/java/1961.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        
        int p, q, cnt = 0, i, dif;

        p = entrada.nextInt();
        q = entrada.nextInt();

        int[] jump = new int[q];

        for(i = 0; i < q; i++)
            jump[i] = entrada.nextInt();

        for(i = 1; i < q; i++){
            if(jump[i] > jump[i-1])
                dif = (jump[i] - jump[i-1]);
            else
                dif = (jump[i-1] - jump[i]);
            if(dif <= p)
                cnt++;
        }
        
        if(cnt == q-1)
            System.out.printf("YOU WIN%n");
        else
            System.out.printf("GAME OVER%n");
    }
}
